src/Centreon/Application/Webservice/TopologyWebservice.php

```diff
@@ -188,10 +189,39 @@ public function getNavigationList()
             ->getRepository(TopologyRepository::class)
             ->getTopologyList($user);
 
+        if ($pollerWizardEntity = $this->determinePollerWizardAccess($user)) {
+            $dbResult[] = $pollerWizardEntity;
+        }
+
         $status = true;
         $navConfig = $this->getDi()[ServiceProvider::YML_CONFIG]['navigation'];
         $result = new NavigationList($dbResult, $navConfig);
 
         return new Response($result, $status);
     }
+
+    /**
+     * @param \CentreonUser $user
+     * @return Topology|null
+     */
+    private function determinePollerWizardAccess(\CentreonUser $user): ?Topology
```

Comment:
The name of the method does not reflect what it does.
Use two methods: one to determine and one to create